website/src/blogs/2019-01-30-performance-tuning-cas-and-openebs.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Performance tuning, CAS and OpenEBS
+author: Evan Powell
+author_info: Founding CEO of a few companies including StackStorm (BRCD) and Nexenta — and CEO & Chairman of OpenEBS/MayaData. ML and DevOps and Python, oh my!
+date: 30-01-2019
+tags: Cassandra, Kubectl, Kubernetes, Mayaonline, OpenEBS, Uncategorized
+excerpt: Now and again — ok almost every day — we get a question on the community or from our commercial users about performance.
+---
+
+Now and again — ok almost every day — we get a question on the community or from our commercial users about performance.
+![IOPS SCREENS FROM MAYAONLINE](/public/images/blog/1.png)
+And our typical response is “mileage may vary.” Sometimes we also mention that there are a bunch of tunables then talk to us for help. And by the way — we love to discuss performance requirements as it shows off how CAS is different and also we get to learn much more about how OpenEBS is being used.
+
+This blog shares some high-level points and then shares info.
+
+First, it is worth noting that Container Attached Storage — where the workload is married to the storage controller — is by definition **`per workload storage`**. So when you compare OpenEBS to, for example, CEPH or some other pre-Kubernetes antediluvian scale-out storage system you risk apples to banana comparisons. While CAS will be constrained by the environment itself and how many resources OpenEBS utilizes from Kubernetes — it will not be directly impacted by the addition of more and more workloads. Those workloads are more or less an island as to themselves. By comparison, a system that checks with a metadata store to determine where to write and where to read the data and stripes data across nodes, for example, will have far greater performance impacts as more workloads are added.
```

Comment:
```suggestion
First, it is worth noting that Container Attached Storage — where the workload is married to the storage controller — is by definition **` per workload storage`**. So when you compare OpenEBS to, for example, CEPH or some other pre-Kubernetes antediluvian scale-out storage system you risk apples to banana comparisons. While CAS will be constrained by the environment itself and how many resources OpenEBS utilizes from Kubernetes — it will not be directly impacted by the addition of more and more workloads. Those workloads are more or less an island as to themselves. By comparison, a system that checks with a metadata store to determine where to write and where to read the data and stripes data across nodes, for example, will have far greater performance impacts as more workloads are added.
```